### PR TITLE
Add logo to login form

### PR DESCRIPTION
**Pull request by lukaw3d**
| Before | After |
| --- | --- |
|  Screenshot from 2024-04-24 23-09-25 |  Screenshot from 2024-04-24 23-07-14 |

### CODE REVIEW

**Review by buberdds — COMMENTED** [review]
something is wrong with a layout 

Screenshot from 2024-04-25 10-56-02

**Comment by lubej** [comment]
> something is wrong with a layout
> 
> Screenshot from 2024-04-25 10-56-02

Yes, `max-width` being set on the title and paragraph for some reason.

**Comment by buberdds** [comment]
The reason is Grommet. It is adding max-width to many components out of a box. We will have to use `fill` prop in Paragraph and Header.

**Comment by lukaw3d** [comment]
Fixed (added fill, but also limited modal width, otherwise it gets super wide)

| 1 | 2 | 3 |
| --- | --- | --- |
| localhost_3000_create-wallet  | localhost_3000_create-wallet (1)  |  localhost_3000_create-wallet (2) |

File: src/app/components/Persist/LoginModalLayout.tsx [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
+  const { dark } = useContext<any>(ThemeContext)
+
   return (
     <Box direction="row-responsive" background="background-back" fill style={{ minHeight: '100dvh' }}>
       <Navigation />
       <ResponsiveLayer
         modal
+        responsive={false}
+        style={{
+          width: 'min(100%, 500px)',
```
Comment (lubej) [inline src/app/components/Persist/LoginModalLayout.tsx:25]: Can we avoid inline styles, and use styled-components instead?

Also don't see how this is different from:

```css
width: 100%
max-width: 500px;
```

Make is much easier to read.

File: src/app/components/Persist/LoginModalLayout.tsx [hunk 2]
```diff
@@ -1,26 +1,43 @@
...
         onClickOutside={props.onClickOutside}
         onEsc={props.onEsc}
       >
         <Box pad="medium">
-          <Header level={2} textAlign="center" margin={{ top: 'medium' }}>
+          <Box align="center" margin={{ vertical: 'large' }}>
+            <img
+              alt="ROSE Wallet logo"
+              src={dark ? walletWhiteLogotype : walletBlueLogotype}
+              style={{ height: '65px' }}
```
Comment (lubej) [inline src/app/components/Persist/LoginModalLayout.tsx:36]: Avoid inline styles. Also, logo looks a bit large to me. Can we scale it down a bit?

Compared to logo in the navigation bar:
image

**Review by lubej — APPROVED** [review]

File: src/app/components/Persist/LoginModalLayout.tsx (hunk 1 above)
Comment (lukaw3d) [reply src/app/components/Persist/LoginModalLayout.tsx:25]: Inline styles:
I would refactor if it's reusable

max-width:
Oh, yep, that does work
(I started from https://github.com/oasisprotocol/oasis-wallet-web/blob/ab5675f07b50e06ba2511d55bf6c9e7c7c98df20/src/app/pages/ParaTimesPage/ParaTimeContent/index.tsx#L23 but that doesn't work correctly in this case)

File: src/app/components/Persist/LoginModalLayout.tsx (hunk 2 above)
Comment (lukaw3d) [reply src/app/components/Persist/LoginModalLayout.tsx:36]: I was looking at old extension, but maybe that isn't intentionally that large: 
image

I'll reduce to 60 :shrug:

**Pull request merged**